packages/block-editor/src/components/iframe/use-scale-canvas.js

```diff
@@ -47,12 +62,12 @@ function computeScrollTopNext( transitionFrom, transitionTo ) {
 		containerHeight: prevContainerHeight,
 		frameSize: prevFrameSize,
 		scaleValue: prevScale,
-		scrollTop,
-		scrollHeight,
+		scrollTop: prevScrollTop,
```

Comment:
Renamed to keep naming consistent with all `transitionFrom` values having a `prev` prefix for clarity.